graph-manager-docs/source/schema-validation.mdx

```diff
@@ -104,7 +268,7 @@ Each of these changes updates an existing schema element. If an operation is act
 
 #### Default arguments
 
-These changes update the default value for an argument. If an operation is using an element of your graph and does not specify a value for this argument, the operation could experience unexpected results when the schema is updated if it was relying on the original default value.
+These change types each update the default value for an argument. If an operation is using an element of your graph and does not specify a value for this argument, the operation might get an unexpected result when the schema is updated if it was relying on the original default value.
```

Comment:
I think the precursor of every sentence being "These change types each" is extremely confusing English. Could we come up with something a bit more readable? Maybe something like "The following all refer to changes in the..." instead?